extensions/wikia/ContributionAppreciation/templates/ContributionAppreciation_diffModule.php

```diff
@@ -0,0 +1,6 @@
+<div class="appreciation-diff-module">
+	<span class="appreciation-cta"><?= wfMessage( 'appreciation-cta-text', $username )->escaped() ?></span>
+	<span class="appreciation-feedback"><?= wfMessage( 'appreciation-feedback-text', $username )->escaped() ?></span>
+	<button class="appreciation-button"
+			data-revision="<?= $revision ?>"><?= wfMessage( 'appreciation-text' )->escaped() ?></button>
```

Comment:
It does, though because the username is also in the request data and is also escaped in the `wfMessage` calls above, it will be double escaped (so `Some "username"` would display as `Some &lt;username&gt;` on viewing the page/module). I'd suggest just escaping in the template where it's needed instead, which is generally a best practice as you should always try to escape as close to the output as possible so it's easier to verify a value is escaped when reviewing.